Cargo.toml

```diff
@@ -30,4 +30,4 @@ logging = ["log"]
 
 [dev-dependencies]
 env_logger = "0.7"
-std-embedded-nal = "=0.0.1"
```

Comment:
Would you mind adding this update to the `CHANGELOG.md` as well? Just add a new section titled "Unreleased" - if not, I'm happy to add it on later :)